lib/iex/lib/iex/autocomplete.ex

```diff
@@ -24,6 +24,14 @@ defmodule IEx.Autocomplete do
     end
   end
 
+  def exports(mod) do
```

Comment:
If we are adding `@doc false` to `exports` do we maybe need add it to `expand` as well?